src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -901,7 +747,7 @@ private UpdateSite.Data scanWAR(
      */
     public static String getMavenModule(
             String plugin, File pluginPath, MavenRunner runner, MavenRunner.Config mavenConfig)
-            throws PomExecutionException, IOException {
+            throws PomExecutionException {
```

Comment:
Again, turning `IOException` into an `UncheckedIOException` right away to prevent it from propagating and for every method to have to `throw IOException` (which only encourages the abuse of `IOException` we see elsewhere).